File: src/main/java/com/procurement/orchestrator/domain/Rule.java
package com.procurement.orchestrator.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Rule {

    private String country;

    private String pmd;

    private String processType;

    private String prevStage;

    private String newStage;

    private String prevPhase;

    private String newPhase;

    private String operationType;
}

File: src/main/kotlin/com/procurement/orchestrator/application/model/Rule.kt
package com.procurement.orchestrator.application.model

import com.procurement.orchestrator.application.model.process.OperationTypeProcess
import com.procurement.orchestrator.application.model.process.ProcessDefinitionKey
import com.procurement.orchestrator.domain.model.ProcurementMethodDetails
import com.procurement.orchestrator.domain.model.address.country.CountryId

class Rule(
    var countryId: CountryId,
    val pmd: ProcurementMethodDetails,
    val processDefinitionKey: ProcessDefinitionKey,
    val stageFrom: Stage?,
    val stageTo: Stage,
    val phaseFrom: Phase?,
    val phaseTo: Phase,
    val operationType: OperationTypeProcess
)
